pkg/virt-handler/hotplug-disk/mount_test.go

```diff
@@ -281,16 +282,17 @@ var _ = Describe("HotplugVolume", func() {
 			Expect(err).ToNot(HaveOccurred())
 
 			record = &vmiMountTargetRecord{}
-
+			kubeletPodDir = "/tmp/kubelet"
```

Comment:
```suggestion
```

Since you do not create this directory explicitly and the tests do not complain, I would assume that in fact it is not used. In such case, could you instead pass an empty string `""` where needed? Just to make it clear.